src/SamplesApp/SamplesApp.UWP.Design/SamplesApp.UWP.Design.csproj

```diff
@@ -131,9 +131,7 @@
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
       <Version>6.2.8</Version>
     </PackageReference>
-    <PackageReference Include="Newtonsoft.Json">
-      <Version>12.0.3</Version>
-    </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Let's change this globally in the Directory.Build.targets

Reply:
Done